--- a/include/reactphysics3d/configuration.h
+++ b/include/reactphysics3d/configuration.h
@@ -29,6 +29,7 @@
 // Libraries
 #include <limits>
 #include <cfloat>
+#include <cstdint>
 #include <utility>
 #include <sstream>
 #include <string>
--- a/include/reactphysics3d/utils/DefaultLogger.h
+++ b/include/reactphysics3d/utils/DefaultLogger.h
@@ -36,6 +36,8 @@
 #include <sstream>
 #include <iomanip>
 #include <mutex>
+#include <ctime>
+#include <chrono>
 
 /// ReactPhysics3D namespace
 namespace reactphysics3d {
